src/smc-webapp/project_new.cjsx

```diff
@@ -271,6 +274,25 @@ ProjectNewForm = rclass ({name}) ->
             on_error     : on_error
             switch_over  : true
 
+    decline_file : ->
+        @setState(warning : false)
+        ReactDOM.findDOMNode(@refs.project_new_filename).disabled = false
```

Comment:
Hmm, I think you can delete that line. One core idea of react is, that such attribute state data is wrapped in components. I.e. that "project new filename" already has a `disabled    = @state.warning` property. Setting it right here directly isn't good. I.e. the setState from above should be enough, right?